README.md

```diff
@@ -1,103 +1,14 @@
-# 100-temporal-mistakes
+# 100 Temporal Mistakes
 
 A collection of common mistakes made when using [Temporal](https://temporal.io) and how to avoid them. This is a work in progress!
 
-## Workflow Limits
-- Overflowing workflow history length
-- Overflowing workflow history bytes
-- Overflowing maximum individual payload size
-- Shard contention due to concurrent updates
-	- Concurrent activities
-	- Concurrent heartbeats
-	- Concurrent child workflows
-	- Frequent signals, updates, queries
-- Exceeding 10s task timeout
+## 1. Workflow Limits
 
-## Workflow Replay
-- Not using workflow versioning/patching
-- Incorrect workflow patching (wrong version numbers, removing branch before it's safe)
-- Thinking that replay means re-running activities
-- Performing network calls in workflow code
-- Using system time instead of workflow time
-- Reading environment variables in workflow code
-- Modifying shared state in workflow code
-- Performing expensive computation in workflow code
-- Passing too much information from activities to workflow code
-- Lossy payload serialization/deserialization
-- Breaking changes to payloads
-- Not using the return value in a side effect
-- Not using the Temporal SDK for logging, metrics, tracing
-- Not using workflow replay for debugging
-- Not using static analysis / sandboxed SDK
-- Modifying workflow history in interceptors or shared libraries
```

Comment:
Can we retain the existing items that don't have written explanations yet? That way we can still go to the index to see what needs to be written (they'll just be plain text instead of links).

Reply:
Sorry I missed the last sentence of the PR description. 😅 

Still I feel like it would be nice to have a single source of truth for the overall list, without the need to cross-reference `MISTAKES.md` with `README.md`. There's also value in seeing the whole list by default when you open the repo, even if there aren't full explanations ready for every entry.

Reply:
Moved things back.